package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "packageManager": "pnpm@9.4.0+sha512.f549b8a52c9d2b8536762f99c0722205efc5af913e77835dbccc3b0b0b2ca9e7dc8022b78062c17291c48e88749c70ce88eb5a74f1fa8c4bf5e18bb46c8bd83a",
```

Comment:
Should I roll this back? I had this version installed, so it was changed automatically.

Reply:
All good 👍🏻